giraffe/src/utils/lineData.perf.test.ts

```diff
@@ -0,0 +1,45 @@
+import {DomainLabel} from '../types'
+import {getDomainDataFromLines} from './lineData'
+import {lineTransform} from '../transforms/line'
+
+import {largeTable, lineData} from './fixtures/line'
+
+// Baseline: one million data points within 3 seconds
+//   this seems arbitrary and will need adjusting
+const MAX_DURATION = 3000
+
+describe('line graph performance', () => {
+  let start, end
+
+  beforeEach(() => {
+    start = Date.now()
+  })
+
+  afterEach(() => {
+    end = Date.now()
+    expect(end - start).toBeLessThanOrEqual(MAX_DURATION)
```

Comment:
we should remove this whole test. performance tests that are set up like this have a weird tendency to slowly approach `MAX_DURATION` as the code... matures?.. and then turn flakey as the test runners don't always have the exact amount of resources / quantum dot spins / star alignments as the previous runs, causing some to run slower and some to run faster.

Reply:
Done. Timing expectations removed. We'll settle for just expecting them to handle large data sets without throwing an error.